File: frontend/src/services/meetingRoom.js
/*
 * @Author: chenrf
 * @Date: 2021-04-06 13:51
 */
import request from '@/utils/request';
import {backendUrl} from './globalconfig'


export async function getClientIp() {
    return request(`${backendUrl}/meeting/client-ip`);
}

File: backend/src/main/java/com/quantil/webrtc/signal/bean/MeetingRoom.java
package com.quantil.webrtc.signal.bean;

import com.quantil.webrtc.api.v1.meeting.bean.RtcMeetingItem;
import lombok.Data;

import java.util.*;

/**
 * @author chenrf
 * @version 1.0
 * @date 2021/4/28 17:47
 */
@Data
public class MeetingRoom {
    private RtcMeetingItem rtcMeetingItem;
    private List<String> admin; // 通过password获取到管理员权限的用户名称
    private String speaker;     // 当前演讲者

    List<MeetingMember> members; // 当前成员信息
    public MeetingRoom(){
        members = new ArrayList<>();
    }
}
